x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/details_page/details_page_mappings_content.tsx

```diff
@@ -237,6 +238,7 @@ export const DetailsPageMappingsContent: FunctionComponent<{
               .map((field) => field.inference_id)
               .filter(
                 (inferenceId: string) =>
+                  !isEmpty(inferenceId) &&
```

Comment:
You can just do `!inferenceId`, no need to import a lodash function that checks way more than is necessary.